packages/venia-concept/src/components/MiniCart/__tests__/product.spec.js

```diff
@@ -7,15 +7,25 @@ import Section from '../section';
 
 configure({ adapter: new Adapter() });
 
-const classes = { firstSection: 'a' };
+const classes = { firstSection: 'a', optionLabel: 'b', name: 'c' };
 
 const item = {
     item_id: 1,
     name: 'Product 1',
     price: 10,
     qty: 1,
     sku: 'TEST1',
-    image: 'test.jpg'
+    image: 'test.jpg',
+    options: [
```

Comment:
> Since these options will be present on the `item` because we merged them from `totals` in the data-layer, you can remove `totalsItems` from `product.js` entirely.

As a corollary to this, we can remove all references to `totalsItems` from this file too 👍